examples/basic_mysql_connection/index.js

```diff
@@ -4,17 +4,28 @@ var mysql = require('mysql');
 var config = JSON.parse(fs.readFileSync("config-secret.json"))
 
 var connection = mysql.createConnection({
-	host: config.host,
+	 host: config.host,
 	user: config.user,
 	password: config.password,
 	port: config.port,
-    database: config.database
+  database: config.database
 });
 
 connection.connect();
+const sqlStatments=['SELECT COUNT(*) as "Total No of Todos" FROM todos',
+					'SELECT * FROM todos WHERE Done=1',
```

Comment:
It is tinyint it can be (-128 to 127 and null), but usualy we define the value and what does it neans.
for exambel if the value =1 it means done and if value =0 means not done.